src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tBody = document.querySelector('tBody');
+const rows = document.querySelectorAll('tBody tr');
+
+const arrayOf = [...rows];
+
+headers.forEach((header) => {
+  header.addEventListener('click', function () {
+    const headerIndex = event.target.cellIndex;
```

Comment:
You are using the global event object here, which is not recommended. Instead, you should pass the event as an argument to the event handler.